### PR TITLE
Export FreeRTOS task-related constants to header file.

### PR DESCRIPTION
**Pull request by Harvie**
This PR exports useful defines to header file.
For more details see: https://github.com/cesanta/mongoose-os/issues/554

### CODE REVIEW

File: include/mgos_freertos.h [hunk 1]
```diff
@@ -23,8 +23,35 @@
...
+
+#ifndef MGOS_MONGOOSE_MAX_POLL_SLEEP_MS
+#define MGOS_MONGOOSE_MAX_POLL_SLEEP_MS 1000
+#endif
+
+#ifndef MGOS_EARLY_WDT_TIMEOUT
+#define MGOS_EARLY_WDT_TIMEOUT 30 /* seconds */
+#endif
+
+#if CS_PLATFORM == CS_P_ESP32
```
Comment (rojer) [inline include/mgos_freertos.h:49]: leave this in the source file, it not really tunable.

Comment (Harvie) [reply include/mgos_freertos.h:49]: Are you talking about the MGOS_EARLY_WDT_TIMEOUT or CS_PLATFORM ?

It's not meant to be tuned... I just want to be able to know the value...

Comment (rojer) [reply include/mgos_freertos.h:49]: STACK_SIZE_UNIT. it's not really a tunable, it is hardcoded in freertos port and we just have to deal with it.

Comment (Harvie) [reply include/mgos_freertos.h:49]: CS_PLATFORM is needed to determine STACK_SIZE_UNIT, which i need.
But i don't really need MGOS_EARLY_WDT_TIMEOUT, but is there reason to conceal it?

Comment (rojer) [reply include/mgos_freertos.h:49]: ah, you need STACK_SIZE_UNIT for your code... fair enough.
then let's keep it public, but please rename it to MGOS_TASK_STACK_SIZE_UNIT

Comment (Harvie) [reply include/mgos_freertos.h:49]: In my module i have code similar to this from `mgos_freertos.c` and it does need STACK_SIZE_UNIT see:

```C
xTaskCreateStatic(mgos_task, "mgos",
                   MGOS_TASK_STACK_SIZE_BYTES / STACK_SIZE_UNIT, NULL,
                    MGOS_TASK_PRIORITY, mgos_task_stack, &mgos_task_tcb);  
```

Comment (Harvie) [reply include/mgos_freertos.h:49]: > please rename it to MGOS_TASK_STACK_SIZE_UNIT

should have seen this coming :) i have pushed fixed version now.

**Comment by rojer** [comment]
thanks!

**Comment by Harvie** [comment]
Thank you too.

**Pull request merged**